_i18n/en/about.md

```diff
@@ -0,0 +1,34 @@
+
+At the same time: don't worry. We'll be open to a new feature, option or preference - whenever there's a case to make for offering our users flexibility. We're working on tags for podcasts, and recently introduced local feeds, for example. So if you've got your own idea, feel free to [share your feature request on our forum](https://forum.antennapod.org/c/feature-request)!
+
+# Open Source and free of ads
+Like other great software such as Firefox and Signal, AntennaPod is open source. This means anyone can inspect and contribute to the code behind the app. And that AntennaPod is availble in the open source and privacy-friendly [F-Droid app store](https://www.f-droid.org/packages/de.danoeh.antennapod/).
+
+That way, you can rest assured (or check for yourself) that there's no shady data collection going on. And because AntennaPod is created by volunteers, we don't need advertisers that bother you while navigating the app.
+
+# Keeping the podcasting ecosystem open
+We stand for an open podcasting ecosystem. An ecosystem that
+* supports free speech, 
+* rewards podcasters for their creative work, 
+* protects listeners from inaccessible content in [walled gardens](https://en.wikipedia.org/wiki/Walled_garden_(technology)), and 
+* allows users to use and switch between apps without [vendor lock-in]([vendor lock-in](https://en.wikipedia.org/wiki/Vendor_lock-in)).
+
+In practice, this means you can subscribe to any RSS feed that's out there, that we make it easy for you to (financially) support podcasters, and that you can synchronise, import and export your data.
+
+# We hear you: community & accessibility
+AntennaPod pushes the content in your ears, but we like to hear from you as well! Our community is open to your feedback and your thoughts. As humans we can disagree, but do so respectfully. And we do our best to make the app friendly for everyone. That's why it is possible to hide automatic suggestions and easy to navigate around the app through talkback.
+
+# Est. 2011: the AntennaPod project today
+The [first bit of code](https://github.com/AntennaPod/AntennaPod/commit/c9283f09dced6f156e13675ef4c13ebeb20cb9e5) of AntennaPod was published in 2011 by Daniel Oeh, the original creator of AntennaPod. Since then, the app has improved and our community grown.
+
+Today, AntennaPod brings together a large community of podcast addicts and open source enthusiasts. The app is installed on over 179.000 active devices (a phone or tablet that's been turned on at least once the past 30 days) via Google play. And we suspect thousands of users have installed the app via F-Droid, the privacy-friendly app store.
```

Comment:
I would write "over 180.000". The rounded number looks better (and we are now at 181.000). Should this use a comma instead of a dot (`180,000`)?